### ISSUE TITLE
Test 230313.017.2bt

### ISSUE THREAD

**Issue by sailist** [pytorch-lumo/image-classification#2]
|Experiment Name|Test Name|Start| End |
|---|---|---|---|
|simclr.simclrexp|230313.017.2bt|23-03-13-174202||

```bash
python3 train_ssl.py --module=simclr --device=1 --config=config/ssl/simclr/cifar10.yaml --model=wrn282 --scan=ssl-2023.02.28
```

## Parameters
```python
{'apply_mixco': False,
 'apply_unmix': False,
 'config': 'config/ssl/simclr/cifar10.yaml',
 'dataset': 'cifar10',
 'detach_cls': True,
 'device': 1,
 'ema': True,
 'ema_alpha': 0.99,
 'epoch': 1000,
 'eval': {'batch_size': 512,
          'num_workers': 8,
          'pin_memory': True,
          'shuffle': True},
 'feature_dim': 128,
 'hidden_feature_size': 128,
 'knn': True,
 'knn_k': 200,
 'knn_t': 0.1,
 'linear_eval': False,
 'lr_decay_end': 0.0005,
 'method': 'simclr',
 'model': 'wrn282',
 'module': 'simclr',
 'more_sample': True,
 'n_classes': 10,
 'optim': {'lr': 0.06, 'momentum': 0.9, 'name': 'SGD', 'weight_decay': 0.0005},
 'pretrain_path': None,
 'scan': 'ssl-2023.02.28',
 'seed': 1,
 'semi_eval': False,
 'stl10_unlabeled': True,
 'temperature': 0.1,
 'test': {'batch_size': 512,
          'num_workers': 8,
          'pin_memory': True,
          'shuffle': False},
 'train': {'batch_size': 512,
           'num_workers': 8,
           'pin_memory': True,
           'shuffle': True},
 'train_ending': 10,
 'train_linear': True,
 'train_strategy': 'ending',
 'warmup_epochs': 0,
 'warmup_from': 0.01,
 'with_bn': False}
```

## Metrics

|Key|Value|
|--|--|
|Acc|7950.00000|
|Knn|82.05000|

Tensor metrics:
```

```

```

```

## Full properties
<details>
<summary> Click Here </summary>
<code>
{'agent': nan,
 'backup': {'23-03-17-161650': {'backend': 'github',
                                'number': 1,
                                'repo': 'pytorch-lumo/image-classification'},
            '23-03-17-161842': {'backend': 'github',
                                'number': 2,
                                'repo': 'pytorch-lumo/image-classification'}},
 'dataset': 'cifar10',
 'deprecated': nan,
 'end': nan,
 'exception': nan,
 'execute': {'cwd': '~/python/image-classification-private',
             'exec_argv': ['train_ssl.py',
                           '--module=simclr',
                           '--device=1',
                           '--config=config/ssl/simclr/cifar10.yaml',
                           '--model=wrn282',
                           '--scan=ssl-2023.02.28'],
             'exec_bin': '~/miniconda3/bin/python3',
             'exec_file': 'train_ssl.py',
             'repo': '~/python/image-classification-private'},
 'exp_name': 'simclr.simclrexp',
 'git': {'commit': '297413b5',
         'dep_hash': '404fc6044b2119d56a5e8b92ac02fc1c',
         'repo': '~/python/image-classification-private'},
 'hooks': {'Diary': {'loaded': True, 'msg': ''},
           'FinalReport': {'loaded': True, 'msg': ''},
           'GitCommit': {'loaded': True, 'msg': ''},
           'LastCmd': {'loaded': True, 'msg': ''},
           'LockFile': {'loaded': True, 'msg': ''},
           'RecordAbort': {'loaded': True, 'msg': ''}},
 'lock': {'accelerate': '0.16.0',
          'decorator': '5.1.1',
          'fire': '0.5.0',
          'hydra': '1.3.1',
          'joblib': '1.2.0',
          'lumo': '0.15.0',
          'numpy': '1.24.2',
          'omegaconf': '2.3.0',
          'psutil': '5.9.4',
          'torch': '1.8.1+cu101',
          'torch.version.cuda': '10.1'},
 'logger_args': {'log_dir': .lumo/experiments/simclr.simclrexp/230313.017.2bt/l.0.2303131742.log'},
 'metric_board': .lumo/blob/simclr.simclrexp/230313.017.2bt/board.sqlite/metric_board.sqlite',
 'model': 'wrn282',
 'note': '',
 'params': {'apply_mixco': False,
            'apply_unmix': False,
            'config': 'config/ssl/simclr/cifar10.yaml',
            'dataset': 'cifar10',
            'detach_cls': True,
            'device': 1,
            'ema': True,
            'ema_alpha': 0.99,
            'epoch': 1000,
            'eval': {'batch_size': 512,
                     'num_workers': 8,
                     'pin_memory': True,
                     'shuffle': True},
            'feature_dim': 128,
            'hidden_feature_size': 128,
            'knn': True,
            'knn_k': 200,
            'knn_t': 0.1,
            'linear_eval': False,
            'lr_decay_end': 0.0005,
            'method': 'simclr',
            'model': 'wrn282',
            'module': 'simclr',
            'more_sample': True,
            'n_classes': 10,
            'optim': {'lr': 0.06,
                      'momentum': 0.9,
                      'name': 'SGD',
                      'weight_decay': 0.0005},
            'pretrain_path': None,
            'scan': 'ssl-2023.02.28',
            'seed': 1,
            'semi_eval': False,
            'stl10_unlabeled': True,
            'temperature': 0.1,
            'test': {'batch_size': 512,
                     'num_workers': 8,
                     'pin_memory': True,
                     'shuffle': False},
            'train': {'batch_size': 512,
                      'num_workers': 8,
                      'pin_memory': True,
                      'shuffle': True},
            'train_ending': 10,
            'train_linear': True,
            'train_strategy': 'ending',
            'warmup_epochs': 0,
            'warmup_from': 0.01,
            'with_bn': False},
 'params.yaml': .lumo/experiments/simclr.simclrexp/230313.017.2bt/params.yaml',
 'params_hash': '099036f350558873c499107c58812d18',
 'paths': {'blob_root': .lumo/blob',
           'cache_root': '/home/share/yanghaozhe/.cache/lumo',
           'info_root': .lumo/experiments'},
 'pinfo': {'hash': '6b82474970b719d71e3548076584f8a2',
           'obj': {'argv': ['~/miniconda3/bin/python3',
                            'train_ssl.py',
                            '--module=simclr',
                            '--device=1',
                            '--config=config/ssl/simclr/cifar10.yaml',
                            '--model=wrn282',
                            '--scan=ssl-2023.02.28'],
                   'pid': 30197,
                   'pname': 'python3',
                   'pstart': 1678700520.57},
           'pid': 30197},
 'progress': {'finished': False,
              'last_edit_time': '23-03-13-174202',
              'ratio': 1.0,
              'start': '23-03-13-174202',
              'update_from': None},
 'rerun': nan,
 'start': nan,
 'state': nan,
 'table_row': .lumo/experiments/simclr.simclrexp/230313.017.2bt/metric.pkl',
 'tags': nan,
 'tensorboard_args': nan,
 'test_name': '230313.017.2bt',
 'trainer': nan}
</code>
</details>
